students/18th/team1/sehyeongkim/user/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from user.views import SignUpView, LoginView
+
+urlpatterns = [
+        path('/signup',SignUpView.as_view()),
+        path('/login', LoginView.as_view()),
+        ]
```

Comment:
indent 가 지금 8칸인데 4칸으로 통일시켜주세요!
```suggestion
urlpatterns = [
        path('/signup',SignUpView.as_view()),
        path('/login', LoginView.as_view()),
]
```